iocage/lib/ioc_fetch.py

```diff
@@ -171,18 +174,24 @@ def fetch_release(self, _list=False):
             except OSError as err:
                 raise RuntimeError(f"{err}")
 
-            if os.path.isdir(f"{self.iocroot}/download/{self.release}"):
+            dataset = f"{self.iocroot}/download/{self.release}"
+
+            if os.path.isdir(dataset):
                 pass
             else:
-                su.Popen(["zfs", "create", "-o", "compression=lz4",
-                          f"{self.pool}/iocage/download/"
-                          f"{self.release}"]).communicate()
-            dataset = f"{self.iocroot}/download/{self.release}"
+                self.zpool.create(f"{self.pool}/iocage/download/{self.release}", {
+                        "compression": "lz4",
+                        "mountpoint": dataset
```

Comment:
We don't want to set a mountpoint here. Then the user will not only have to change `POOL/iocage`'s mountpoint (or we do) but they would also have to change every `download/RELEASE`'s mountpoint as well. It's best to let them inherit that from the parent.